docs/book/component-gallery/orchestrators/airflow.md

```diff
@@ -87,6 +87,47 @@ This call will produce a `.zip` file containing a representation of your ZenML
 pipeline to the Airflow DAGs directory. From there, the local Airflow server
 will load it and run your pipeline (It might take a few seconds until the pipeline
 shows up in the Airflow UI).
+
+{% hint style="info" %}
+
+The ability to provision resources using the `zenml stack up` command
```

Comment:
Best to also raise a warning in the code if users try `zenml stack up` with a local airflow component`